### PR TITLE
[ticket/16000] Link to php's date()

### PR DESCRIPTION
**Pull request by senky**
PHPBB3-16000

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16000

### CODE REVIEW

File: phpBB/language/en/acp/board.php [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	'BOARD_STYLE'					=> 'Board style',
 	'CUSTOM_DATEFORMAT'				=> 'Custom…',
 	'DEFAULT_DATE_FORMAT'			=> 'Date format',
-	'DEFAULT_DATE_FORMAT_EXPLAIN'	=> 'The date format is the same as the PHP <code>date</code> function.',
+	'DEFAULT_DATE_FORMAT_EXPLAIN'	=> 'The date format is the same as the PHP <code><a href="https://secure.php.net/manual/function.date.php" target="_blank" rel="noreferrer noopener">date()</a></code> function.',
```
Comment (Crizz0) [inline phpBB/language/en/acp/board.php]: I would prefer to not use "target=_blank" at all. User should be able to use there browser properly...CRLT+click or something.

Comment (3D-I) [reply phpBB/language/en/acp/board.php]: Well, it has its own logic though.

Comment (mrgoldy) [reply phpBB/language/en/acp/board.php]: The link to Jabber and phpBB in the ACP do not have the `target` attribute either.
I think it's best to omit and leave it to the user, on how to open it.
It's within the ACP settings pages, so no posts are losts.

Comment (3D-I) [reply phpBB/language/en/acp/board.php]: https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/

https://mathiasbynens.github.io/rel-noopener/

**Pull request merged**